# Lidar range value from 260째 - 280째 -> Caution median index -> 270째 => ix = [280,...271,270,269,267,...,260]
float32[] Lidar_Range_Values_At_270
# Lidar value at 270째
float32   Lidar_Value_At_270